zerver/views/development/integrations.py

```diff
@@ -42,17 +45,25 @@ def dev_panel(request: HttpRequest) -> HttpResponse:
 
 def send_webhook_fixture_message(
     url: str, body: str, is_json: bool, custom_headers: Dict[str, Any]
-) -> HttpResponse:
+) -> "TestHttpResponse":
     client = Client()
     realm = get_realm("zulip")
+    follow = custom_headers.pop("follow", False)
+    secure = custom_headers.pop("secure", False)
```

Comment:
It’s not possible for `standardize_headers` to return a `follow` or `secure` key. We should hard-code them to `False`.

Reply:
Updated.